### PR TITLE
[HDInsight] Add deprecation info to old private link's arguments

### PR DESCRIPTION
**Pull request by aim-for-better**
Add deprecation info for argument public_networrk_access_type and outbound_only_public_network_access_type

**Description**  

**Testing Guide**  

**History Notes**  

[HDInsight] : az hdinsight create: add deprecate information for argument --public-networrk-access-type and --outbound-public-network-access-type

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
HDInsight

File: src/azure-cli/azure/cli/command_modules/hdinsight/_params.py [hunk 1]
```diff
@@ -184,10 +184,12 @@ def load_arguments(self, _):
...
         # Private Link Network Settings
         c.argument('public_network_access_type', arg_group='Private Link Network Settings',
-                   arg_type=get_enum_type(PublicNetworkAccess), help='The public network access type.')
+                   arg_type=get_enum_type(PublicNetworkAccess), help='The public network access type.',
+                   deprecate_info=c.deprecate(expiration='2.12.2'))
         c.argument('outbound_public_network_access_type', arg_group='Private Link Network Settings',
                    arg_type=get_enum_type(OutboundOnlyPublicNetworkAccessType),
-                   help='The outbound only public network access type.')
+                   help='The outbound only public network access type.',
+                   deprecate_info=c.deprecate(expiration='2.12.2'))
```
Comment (aim-for-better) [inline src/azure-cli/azure/cli/command_modules/hdinsight/_params.py:192]: The version should be the same with S177's version. What is the version in S177?

Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/hdinsight/_params.py:192]: You mean sprint 177?

**Review by haroldrandom — CHANGES_REQUESTED** [review]
You mean sprint 177? If so, then it should be 2.14.0

**Comment by aim-for-better** [comment]
> You mean sprint 177? If so, then it should be 2.14.0



> You mean sprint 177? If so, then it should be 2.14.0

Hi @haroldrandom Thanks, yes I mean the S177, I have changed the version to `2.14.0`.

**Review by haroldrandom — APPROVED** [review]

**Pull request merged**